.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+  updates:
```

Comment:
There's an extra indentation that messes everything up. Did the same mistake earlier

Reply:
Updated